backend/src/judge/judge.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable } from "@nestjs/common";
+import axios from "axios";
+import { AxiosRequestConfig } from "axios";
+import { InjectPinoLogger, PinoLogger } from "nestjs-pino";
+
+import { ConfigService } from "src/core/config/config.service";
+
+import { Language } from "~shared/types/base/index";
+
+@Injectable()
+export class JudgeService {
+  constructor(
+    @InjectPinoLogger(JudgeService.name)
+    private readonly logger: PinoLogger,
+    private configService: ConfigService,
+  ) {}
+
+  async sendRequest(
+    language: Language,
+    code: string,
+    stdin: string,
+    expectedOutput: string,
+  ): Promise<boolean> {
+    this.logger.info("Sending code to Judge0...");
+    const options: AxiosRequestConfig<string> = {
+      method: "POST",
+      url: `https://${this.configService.get("judge0.apiHost")}/submissions`,
+      // TODO: Use "wait: false" and get submission status with token
```

Comment:
As an aside, we will likely provide them a callback URL to our API that will give us the results, which we can then get to the user through a websocket connection of some sort.